RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -39,4 +42,17 @@
                             ConvStepTask,
                             conversionStepTracks
                             )
+
+iterTrackingTask_trackdnn = cms.Task(tf_dummy_source,
+                            InitialStepPreSplittingTask,
+                            trackerClusterCheck,
+                            iterTrackingEarlyTask,
+                            earlyGeneralTracks,
+                            muonSeededStepTask,
+                            preDuplicateMergingGeneralTracks,
+                            generalTracksTask,
+                            ConvStepTask,
+                            conversionStepTracks
+                            )
```

Comment:
this may technically run, but copy-pasting the existing task is not a good solution, as now both have to be kept in sync. Instead, you can use the copy-and-add pattern, as is done for example here: https://github.com/cms-sw/cmssw/blob/caa48369fc0e28abc8536b955ddb93348b660cd0/Configuration/StandardSequences/python/Reconstruction_cff.py#L123

In addition, stuff defined in this config that **should not** be imported elsewhere should start with an underscore, this is the default unless in special cases

```suggestion
_iterTrackingTask_trackdnn = iterTrackingTask.copy()
_iterTrackingTask_trackdnn.add(_trackdnn_source)
```